build-tools/packages/build-cli/src/library/changesets.ts

```diff
@@ -239,21 +239,21 @@ export async function loadChangesets(dir: string, log?: Logger): Promise<Changes
 		// eslint-disable-next-line no-await-in-loop
 		const rawFileContent = await readFile(file, { encoding: "utf8" });
 
-		// Parse out the first layer of metadata, which is the package --> change type mapping.
-		const firstParse = matter(rawFileContent);
-		const packageBumpTypeMetadata = firstParse.data;
-
-		// If there is a second frontmatter section, parse it as the additional metadata.
-		const hasAdditionalMetadata = hasFrontMatter(firstParse.content);
```

Comment:
This seems to have been the only use of `hasFrontMatter`... build-tools doesn't complain about unused imports?

Reply:
Yeah, looks like the tsconfig option is disabled but the lint rule is not enabled. I'll follow up with another change to enable the lint rule and fix violations.

Reply:
Follow up PR: #24272